cli/clustermesh.go

```diff
@@ -243,7 +243,7 @@ func newCmdClusterMeshEnableWithHelm() *cobra.Command {
 			params.HelmReleaseName = helmReleaseName
 			ctx := context.Background()
 			if err := clustermesh.EnableWithHelm(ctx, k8sClient, params); err != nil {
-				fatalf("Unable to enable ClusterMesh: %s", err)
+				fatalf("Unable to enable ClusterMesh: %s, please ensure that you have specified the --namespace and --helm-release-name if you are working with a cluster with customized Cilium installation.", err)
```

Comment:
Might be an easier to grok if these read something like `failed to enable/disable xxx`

Reply:
good point, also I did mistake by starting it with uppercase

Reply:
I will also include other error messages

Reply:
fatalf is just a wrapper to print a message to stderr and exit. Thus messages can start with upper case.

Reply:
correct, I restored it